examples/turbulent_advection-diffusion.jl

```diff
@@ -148,7 +151,7 @@ while clock.step <= nsteps
   MultiLayerQG.updatevars!(params.MQGprob)
 end
 
```

Comment:
Add `nothing # hide` at the end of this loop?

Reply:
no need here